File: POCAppJava/app/src/main/java/com/poc/pocappjava/model/FeedRow.java
package com.poc.pocappjava.model;

import com.google.gson.annotations.SerializedName;

/**
 * Created by akhilmalik on 08/02/18.
 */

public class FeedRow {
    @SerializedName("title")
    String title;
    @SerializedName("description")
    String description;
    @SerializedName("imageHref")
    String image;

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public String getImage() {
        return image;
    }

    public void setImage(String image) {
        this.image = image;
    }
}

File: app/src/main/java/com/hcl/poc/model/FeedRow.kt
package com.hcl.poc.model

import com.google.gson.annotations.SerializedName

/**
 * Created by akhilmalik on 07/02/18.
 *
 * Data class for Feed Rows
 */
data class FeedRow(
        @SerializedName("title")
        val title: String,
        @SerializedName("description")
        val description: String,
        @SerializedName("imageHref")
        val image: String

) {
}